eng/pipelines/templates/jobs/azuresdkpartnerdrops-to-devops.yml

```diff
@@ -0,0 +1,99 @@
+resources:
+  repositories:
+    - repository: azure-sdk-build-tools
+      type: git
+      name: internal/azure-sdk-build-tools
+      ref: refs/tags/azure-sdk-build-tools_20211215.1
+
+parameters:
+- name: BuildToolsRepoPath
+  type: string
+  default: '$(System.DefaultWorkingDirectory)/_azure-sdk-build-tools'
+- name: ArtifactsPath
+  type: string
+  default: ''
+- name: PartnerDropsBlobPath
+  type: string
+  default: ''
+- name: NugetVersion
+  type: string
+  default: ''
+- name: DevOpsFeedID
+  type: string
+  default: ''
+- name: PartnerDropsSubscription
+  type: string
+  default: ''
+- name: PartnerDropsStorageName
+  type: string
+  default: ''
+- name: PartnerDropsContainerName
+  type: string
+  default: ''
+- name: PartnerDropsBlobPrefix
+  type: string
+  default: ''
+
+jobs:
+  - job: NupkgReleases
+    pool:
+      name: azsdk-pool-mms-win-2019-general
+      vmImage: MMS2019
+
+  steps:
+    - template: tools/clone-buildtools/clone-buildtools.yml@azure-sdk-build-tools
+      parameters:
+        BuildToolsRepoPath: ${{ parameters.BuildToolsRepoPath }}
+
+    - task: PowerShell@2
+      displayName: 'Copy from AzureSdkPartnerDrops'
+      inputs:
+        targetType: filePath
+        filePath: '${{ parameters.BuildToolsRepoPath }}/scripts/copy-from-azuresdkpartnerdrops.ps1'
```

Comment:
I assume we plan to move this script into azure-sdk-tools repo correct?

Reply:
I don't think it should be moved, since it relies on a checked in version of azcopy.exe.
An option will be to retire it and just use the `AzureFileCopy` task instead.

Reply:
Yeah we should probably use an existing file copy task instead of relying directly on azcopy. You can do that as a follow-up.